archinstall/lib/args.py

```diff
@@ -392,6 +393,12 @@ def _define_arguments(self) -> ArgumentParser:
 			default=False,
 			help='Disabled online upstream services such as package search and key-ring auto update.',
 		)
+		parser.add_argument(
```

Comment:
I'm somewhat against introducing a new flag to specifically handle the mirror fetching failures, if this continues we'll have a flag for every menu entry which isn't sustainable and very confusing for users. 

I'd rather recommend to handle this gracefully in the menu and provide the users with options if possible or just fall back to the local config.